### ISSUE TITLE
Are the .on's not emitting?

### ISSUE THREAD

**Issue by laduke** [choojs/nanostate#8]
it seems like the machine.on('x', cb) things aren't happening for me

```
var nanostate = require('nanostate')

var machine = nanostate('idle', {
  idle: { click: 'loading' },
  loading: { resolve: 'data', reject: 'error' },
  data: { click: 'loading' },
  error: { click: 'loading' }
})

console.log(machine.state)
machine.emit('click')
console.log(machine.state)

machine.on('idle', () => console.log('on idle'))
machine.on('loading', () => console.log('on loading'))
```

**Comment by yoshuawuyts** [comment]
Ping @kareniel any idea what might be happening here?

**Comment by kareniel** [comment]
oh, strange. 
i think i have an idea where to look. will give it some time later today!

**Comment by kareniel** [comment]
okay so the events -are- being emitted but we are emitting the original event name (ie: 'click'), not the state name. i checked the docs and it seems to suggest that the latter is the expected behavior.
gonna make a quick pr.

**Comment by laduke** [comment]
thanks for working with my bad example where i send events before i register listeners for them 😅

**Comment by yoshuawuyts** [comment]
Fixed in `v1.2.1` :tada:
